test/e2e/common/pods.go

```diff
@@ -832,7 +832,13 @@ var _ = framework.KubeDescribe("Pods", func() {
 
 	})
 
-	ginkgo.It("should delete a collection of pods", func() {
+	/*
+		Release : v1.19
+		Testname: Pods, delete a collection
+		Description: A set of pods is created with a label selector which MUST be found when listed.
+		The set of pods is deleted and MUST NOT show up when listed by its label selector.
+	*/
+	framework.ConformanceIt("should delete a collection of pods", func() {
```

Comment:
It seems difficult to say this test is stable to become conformance test according to the testgrid result.
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce/1281513716512198658 is an example.

/cc @oomichi

Reply:
Looking at a couple recent fails here:

<img width="870" alt="image" src="https://user-images.githubusercontent.com/31331/87846516-9a1fc380-c924-11ea-87fe-fc278086370d.png">

<img width="730" alt="image" src="https://user-images.githubusercontent.com/31331/87846555-d5ba8d80-c924-11ea-900f-5006351ede43.png">

Will look into this next week.